README.md

```diff
@@ -21,62 +18,58 @@ Skip tests:
-| KAFKA_CONSUMER_MAX_POLL_RECORDS `*`  | 200           | Maximum number of records returned in a single call to poll().                                                                                             |
-
+| Name                                                | Default Value                   | Description                                                                                                                                                                           |
+|-----------------------------------------------------|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ENV                                                 | folio                           | The logical name of the deployment, must be unique across all environments using the same shared Kafka/Elasticsearch clusters, `a-z (any case)`, `0-9`, `-`, `_` symbols only allowed |
+| DB_HOST                                             | localhost                       | Postgres hostname                                                                                                                                                                     |
+| DB_PORT                                             | 5432                            | Postgres port                                                                                                                                                                         |
+| DB_USERNAME                                         | postgres                        | Postgres username                                                                                                                                                                     |
+| DB_PASSWORD                                         | postgres                        | Postgres password                                                                                                                                                                     |
+| DB_DATABASE                                         | postgres                        | Postgres database name                                                                                                                                                                |
+| SPRING_PROFILES_ACTIVE                              | -                               | Indicates if the application has to be run in FOLIO mode or standalone mode                                                                                                           |
+| KAFKA_HOST`*`                                       | kafka                           | Kafka broker hostname                                                                                                                                                                 |
```

Comment:
Can we add `server.port` with default value `8081` here?

Reply:
it's not an environment variable, is it?
But we could use a default one provided by Spring: [SERVER_PORT](https://docs.spring.io/spring-boot/how-to/webserver.html#howto.webserver.change-port)
Added, thank you.